demos/python_demos/llm_text_generation/servable_stream/model.py

```diff
@@ -181,6 +181,7 @@ def generate():
         for partial_result in streamer:
             yield serialize_completions(batch_size, partial_result)
         t1.join()
+        token_count[0] -= len(tokens["input_ids"].flatten())
         yield [Tensor("token_count", np.array([token_count[0]], dtype=np.int32))]
```

Comment:
why only 1st element of token_count? Was it tested with bs>1 with variable response size?

Reply:
First element of token_count represents final number of tokens. This snipped uses list instead of single variable due to the mutable property of python's list type.

Reply:
I think @dtrawins misunderstood token_count variable - it is **always** 1 element array in order to return it as numpy array.

But @dtrawins makes good point here - @bstrzele should we track token count *per batch* or just sum of tokens in all of batches?